metrics/Contributing_Organizations.md

```diff
@@ -3,27 +3,24 @@
 Question: What is the number of contributing organizations?

```

Comment:
Perhaps rephrase as "How many organizations are contributing to this project, including both code and issues?"